src/App.tsx

```diff
@@ -1,34 +1,71 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import './App.css'
+import React, { FC, PropsWithChildren, lazy, Suspense } from 'react';
+import { observer } from 'mobx-react-lite';
+import {
+  createBrowserRouter,
+  RouterProvider,
+  Route,
+  createRoutesFromElements,
+} from 'react-router-dom';
+import { ConfigProvider, theme } from 'antd';
+import routePaths from './utils/constants/routePaths';
+import Spinner from './components/Spinner';
+import MainLayout from './layouts/MainLayout';
+import colorTheme from './store/theme';
+import './assets/styles/index.scss';
 
-function App() {
-  const [count, setCount] = useState(0)
+const MainPage = lazy(() => import('./pages/MainPage'));
+const NotFoundPage = lazy(() => import('./pages/NotFoundPage'));
+const ChartsPage = lazy(() => import('./pages/ChartsPage'));
 
-  return (
-    <div className="App">
-      <div>
-        <a href="https://vitejs.dev" target="_blank">
-          <img src="/vite.svg" className="logo" alt="Vite logo" />
-        </a>
-        <a href="https://reactjs.org" target="_blank">
-          <img src={reactLogo} className="logo react" alt="React logo" />
-        </a>
-      </div>
-      <h1>Vite + React</h1>
-      <div className="card">
-        <button onClick={() => setCount((count) => count + 1)}>
-          count is {count}
-        </button>
-        <p>
-          Edit <code>src/App.tsx</code> and save to test HMR
-        </p>
-      </div>
-      <p className="read-the-docs">
-        Click on the Vite and React logos to learn more
-      </p>
-    </div>
+const SuspenseRoute: FC<PropsWithChildren> = ({ children }) => {
```

Comment:
probably we can move all settings and router setup to the separate folder -> router. And then import RouterProvider to the App.tsx